share/scripts/inverse/update_ibi_pot.pl

```diff
@@ -56,17 +56,43 @@
 (readin_table($cur_pot_file,@pot_r_cur,@pot_cur,@pot_flags_cur)) || die "$progname: error at readin_table\n";
 
 #should never happen due to resample, but better check
-die "Different grids \n" if (($r_aim[1]-$r_aim[0]-$r_cur[1]+$r_cur[0])>0.0001);
+die "Different grids \n" if (($r_aim[1]-$r_aim[0]-$r_cur[1]+$r_cur[0]) > 0.0001);
 die "Different start potential point \n" if (($r_aim[0]-$r_cur[0]) > 0.0001);
 die "Different end potential point \n" if ( $#r_aim != $#r_cur );
 
 my $outfile="$ARGV[3]";
 my @dpot;
 my @flag;
-my $value=0.0;
 
-#start from the end to make the begining have the last value
-for (my $i=$#r_aim;$i>=0;$i--){
+sub maxindex {
```

Comment:
Maybe add comments what this function is doing.

If it could useful in more place move it to the cage functions perl module.

Reply:
added a comment. Did not move it.